### DISCUSSION TITLE
`pnpm fund` that works like `npm fund`

### DISCUSSION THREAD

**Discussion by sapphi-red** [pnpm/pnpm#15911] [Ideas]
### Contribution

- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

### Describe the user story

Both npm and yarn has a command called `fund` that retrieves information on how to fund the dependencies of a given project. I want to run that command for all packages in the workspace.

- https://docs.npmjs.com/cli/v10/commands/npm-fund
- https://yarnpkg.com/blog/release/2.1#whats-to-come:~:text=yarn%20list%20%26-,yarn%20fund,-Add%20the%20telemetry

### Describe the solution you'd like

Add `pnpm fund` command.

### Describe the drawbacks of your solution

Adds maintainance burden?

### Describe alternatives you've considered

Not to add the command. I can simply run `npm fund` for all packages in my workspace. But I expect `pnpm fund` to work as it works with npm and yarn.

**Comment by zkochan** [comment]
Sounds good.

**Comment by altaywtf** [comment]
I'm working on a fix for this.

Written by an agent (Claude Code, Opus 5.5).

**Comment by altaywtf** [comment]
A fix is ready for review in https://github.com/pnpm/pnpm/pull/15850. CI is green.

Written by an agent (Claude Code, Opus 5.5).
